### PR TITLE
v6.20.5

### PR DESCRIPTION
**Pull request by KenanYusuf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,29 @@
...
+
+We'd like to offer a big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+
+### Data Grid
+
+#### `@mui/x-data-grid-premium@6.20.5` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+- [DataGridPremium] Fix onCellSelectionModelChange not triggered when additional cell range is selected (#14650) @arminmeh
```
Comment (arminmeh) [inline CHANGELOG.md:18]:
```suggestion
- [DataGridPremium] Fix `onCellSelectionModelChange` not triggered when additional cell range is selected (#14650) @arminmeh
```

**Comment by arminmeh** [comment]
Should other packages get a version bump as well?

**Comment by LukasTy** [comment]
> Should other packages get a version bump as well?

We didn't have such an approach for releases on v6.
Besides, even on v7, we rely on Lerna, if it suggests a bump—we do it.

@KenanYusuf, could you try pushing an empty commit?
I have restarted the Netlify deployment without a cache and it succeeded: https://app.netlify.com/sites/material-ui-x/deploys/66ec0f80fad5db9154cd9c8f

**Review by LukasTy — APPROVED** [review]

**Comment by arminmeh** [comment]
> @KenanYusuf, could you try pushing an empty commit?

I have applied my own suggestion 🙂

**Review by arminmeh — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,29 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.20.5
+
+_Sep 19, 2024_
+
+We'd like to offer a big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+
```
Comment (MBilalShafi) [inline CHANGELOG.md:13]:
```suggestion
- 📚 Documentation improvements

```

File: CHANGELOG.md
Comment (MBilalShafi) [inline CHANGELOG.md:1]: Nitpick on Line 51:
```diff
- ## v6.20.3
+ ## 6.20.3
```

**Review by MBilalShafi — APPROVED** [review]

File: packages/grid/x-data-grid-pro/package.json
Comment (cherniavskii) [inline packages/grid/x-data-grid-pro/package.json:1]: Did lerna bump these? Or is this bump only for version consistency?

Comment (KenanYusuf) [reply packages/grid/x-data-grid-pro/package.json:1]: I forgot we had a script 😅. Found it after reading the release guide again.

Lerna does not bump pro and community, so I'll undo that unless they should actually be bumped for consistency.

Comment (cherniavskii) [reply packages/grid/x-data-grid-pro/package.json:1]: We tend to release whatever lerna suggests, I wouldn't bump other packages in this case 👍🏻

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,30 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.20.5
+
+_Sep 19, 2024_
```
Comment (KenanYusuf) [inline CHANGELOG.md:8]:
```suggestion
_Sep 20, 2024_
```

**Pull request merged**